<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生信息管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Icon/administration.svg</normaloff>:/Icon/administration.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget {
	color: rgb(102, 155, 188);
    background-color: rgb(102, 155, 188); /* 设置背景颜色为 a2d2ff */
}
</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="5,0,0">
       <item row="0" column="0">
        <widget class="QLabel" name="label_XinXiGuanLi">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(65, 90, 119);
font: 16pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>学生信息管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QPushButton" name="btn_TuiChu">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>AcadEref</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(255, 175, 204); /* 背景色 */
    color: white; /* 文字颜色 */
    font-size: 19px; /* 文字大小 */
    border: none; /* 去掉边框 */
    width: 93px; /* 固定宽度 */
    height: 35px; /* 固定高度 */
    border-radius: 10px; /* 圆角 */
}

QPushButton:hover {
    background-color: rgb(251, 111, 146); /* 鼠标悬停颜色变亮 */
}

QPushButton:pressed {
    background-color: rgb(255, 0, 110); /* 按下时颜色变暗 */
    transform: scale(0.95); /* 点击时缩小效果 */
}
</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_TouXiang">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/Icon/TouXiang.JPG);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_YongHuMing">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(65, 90, 119);
font: 12pt &quot;黑体&quot;;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(65, 90, 119);
font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>全选</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ZengJia">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(255, 175, 204); /* 背景色 */
    color: white; /* 文字颜色 */
    font-size: 19px; /* 文字大小 */
    border: none; /* 去掉边框 */
    width: 93px; /* 固定宽度 */
    height: 35px; /* 固定高度 */
    border-radius: 10px; /* 圆角 */
}

QPushButton:hover {
    background-color: rgb(251, 111, 146); /* 鼠标悬停颜色变亮 */
}

QPushButton:pressed {
    background-color: rgb(255, 0, 110); /* 按下时颜色变暗 */
    transform: scale(0.95); /* 点击时缩小效果 */
}</string>
          </property>
          <property name="text">
           <string>增加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_XiuGai">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(255, 175, 204); /* 背景色 */
    color: white; /* 文字颜色 */
    font-size: 19px; /* 文字大小 */
    border: none; /* 去掉边框 */
    width: 93px; /* 固定宽度 */
    height: 35px; /* 固定高度 */
    border-radius: 10px; /* 圆角 */
}

QPushButton:hover {
    background-color: rgb(251, 111, 146); /* 鼠标悬停颜色变亮 */
}

QPushButton:pressed {
    background-color: rgb(255, 0, 110); /* 按下时颜色变暗 */
    transform: scale(0.95); /* 点击时缩小效果 */
}
</string>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ShanChu">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(255, 175, 204); /* 背景色 */
    color: white; /* 文字颜色 */
    font-size: 19px; /* 文字大小 */
    border: none; /* 去掉边框 */
    width: 93px; /* 固定宽度 */
    height: 35px; /* 固定高度 */
    border-radius: 10px; /* 圆角 */
}

QPushButton:hover {
    background-color: rgb(251, 111, 146); /* 鼠标悬停颜色变亮 */
}

QPushButton:pressed {
    background-color: rgb(255, 0, 110); /* 按下时颜色变暗 */
    transform: scale(0.95); /* 点击时缩小效果 */
}
</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="maximumSize">
           <size>
            <width>1000</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    border: 2px solid rgb(255, 175, 204); /* 边框颜色 */
    border-radius: 15px; /* 设置圆角 */
    padding: 8px; /* 内边距 */
    font-size: 15px; /* 字体大小 */
    background-color: rgb(240, 240, 240); /* 背景颜色 */
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_SouSuo">
          <property name="font">
           <font>
            <family>AcadEref</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(255, 175, 204); /* 背景色 */
    color: white; /* 文字颜色 */
    font-size: 19px; /* 文字大小 */
    border: none; /* 去掉边框 */
    width: 93px; /* 固定宽度 */
    height: 35px; /* 固定高度 */
    border-radius: 10px; /* 圆角 */
}

QPushButton:hover {
    background-color: rgb(251, 111, 146); /* 鼠标悬停颜色变亮 */
}

QPushButton:pressed {
    background-color: rgb(255, 0, 110); /* 按下时颜色变暗 */
    transform: scale(0.95); /* 点击时缩小效果 */
}
</string>
          </property>
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_RenShu">
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(65, 90, 119);
font: 8pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>550</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>550</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
